docs/getting-started/integration_rewards.md

```diff
@@ -8,6 +8,15 @@ permalink: /getting-started/integration-rewards/
 
 # Integration rewards
 
-We encourage you to apply for integration rewards (upto **$20,000**) once your project is
-successfully integrated with OSS-Fuzz. Please see the details in our blog post
+We encourage you to apply for integration rewards (up to **$20,000**) once your project
+is successfully integrated with OSS-Fuzz. Please see the details in our blog post
 [here](https://opensource.googleblog.com/2017/05/oss-fuzz-five-months-later-and.html).
+
+Note that the top rewards are issued to projects that satisfy both of the following
```

Comment:
Should we explicitly state the minimum rewards we give for each category of integration? e.g. X-5000. 

I think maybe this can be worded a little more clearly along the lines of:

Rewards are based on the qualify of integration, using the 4 pooints mentioned in the blog post:
 (repeat the blog post criteria here: coverage, upstream integration etc).

The reward amounts for each point range from X-$5000. The highest thresholds ($5000) are awarded only to projects of a critical importance for the global infrastructure and/or widely used products, devices, or services...

Reply:
Makes sense, done! PTAL :)